suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -39,107 +38,103 @@ const sheetContentContainerStyle = prepareNativeStyle<{
     paddingHorizontal: utils.spacings.sp16,
 }));
 
-const sheetWithOverlayStyle = prepareNativeStyle(_ => ({
-    flex: 1,
-    justifyContent: 'flex-end',
+const handleStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.borderDashed,
 }));
 
+const WindowOverlay = ({ children }: { children: ReactNode }) => {
+    if (Platform.OS === 'ios') {
+        return <FullWindowOverlay>{children}</FullWindowOverlay>;
+    }
+
+    return (
+        <View style={{ position: 'absolute', top: 0, left: 0, right: 0, bottom: 0 }}>
```

Comment:
Move the styles to separated styleSheet object please.

Reply:
fixed